reads_filtration/mapitope_conversion.py

```diff
@@ -0,0 +1,84 @@
+# import datetime
+import sys
+import logging
+import os
+logger = logging.getLogger('main')
+
+
+if os.path.exists('/groups/pupko/orenavr2/'):
+    src_dir = '/groups/pupko/orenavr2/igomeProfilingPipeline/src'
+elif os.path.exists('/Users/Oren/Dropbox/Projects/'):
+    src_dir = '/Users/Oren/Dropbox/Projects/gershoni/src'
+else:
+    src_dir = '.'
+sys.path.insert(0, src_dir)
+
+# needs $src_dir in path
+from auxiliaries.pipeline_auxiliaries import load_table_to_dict
+
+mapitope = {
+    'R': 'B',
+    'K': 'B',
+    'E': 'J',
+    'D': 'J',
+    'S': 'O',
+    'T': 'O',
+    'I': 'U',
+    'L': 'U',
+    'V': 'U',
+    'Q': 'X',
+    'N': 'X',
+    'W': 'Z',
+    'F': 'Z',
+    'A': 'A',
+    'C': 'C',
+    'G': 'G',
+    'H': 'H',
+    'M': 'M',
+    'P': 'P',
+    'Y': 'Y'
+}
+
+def convert_to_mapitope(sequence):
+    return ''.join([mapitope[c] for c in sequence])
+
+
+def convert_faa_file_by_maptope(input_file_path, output_file_path):
+    with open(input_file_path,'r') as input_file, open(output_file_path,'w+') as output_file:
+        input_lines = input_file.readlines()
+        for line in input_lines:
+            if line[0] == '>':
+                output_file.write(line)  
+            else:
+                output_file.write(convert_to_mapitope(line[:-1])+'\n')
+
+def mapitope_convert_all_samples(reads_filtration_path, barcode2samplename_path, done_file_path, argv='no_argv'):
```

Comment:
Can be done in parallel as at this point we have a separate file/directory per sample